docs/install.rst

```diff
@@ -304,11 +304,20 @@ section of ``mexopts.sh``.
 3) Windows: tips for compiling
 -------------------------------   
    
-We have users who have adjusted the makefile to work with Windows. Please
-try::
+We have users who have adjusted the makefile to work - at least to some extent - on Windows 10. Please make sure to have a recent version of Mingw at hand, preferably with a 64bit version of gnu-make like the WinLibs standalone build of GCC and MinGW-w64 for Windows. Note that most MinGW-w64 distributions, such as TDM-GCC, do not feature the 64bit gnu-make. Fortunately, this limitation is only relevant to run the tests. To build the static and dynamic library try::
 
-  cp make.inc.windows_mingw make.inc
-  make test -j
+  copy make.inc.windows_mingw make.inc
+  make lib CPATH=(PATH_OF_FFTW_H) LIBRARY_PATH=(PATH_OF_FFTW(F)_DLL)
+
+Keep in mind that the MinGW installation contains only a file called mingw32-make.exe, not make.exe. Create a copy of this file, call it make.exe, and make sure the parent folder is part of your system PATH. To shorten your make calls, you can also add ``CPATH`` and ``LIBRARY_PATH`` with the required values to your ``make.inc`` instead of appending the assignments to the make command. If the library is compiled successfully, you can try to run the tests. Note that your system has to fulfill the following prerequisites to this end: A Linux distribution set up via WSL (has been tested with Ubuntu 20.04 LTS from the Windows Store) and the 64bit gnu-make mentioned before. Further, make sure that the directory containing the FFTW-DLLs is part of your system PATH. As soon as you have everything set up, run the following command::
+
+  make test CPATH=(PATH_OF_FFTW_H) LIBRARY_PATH=(PATH_OF_FFTW(F)_DLL)
+
+In a similar fashion, the examples can be build by replacing ``make test`` with ``make examples``. This rule of the makefile does neither require WSL or the 64bit gnu-make and should hopefully work out-of-the-box. Finally, it is also possible to build the MEX file needed to call FINUFFT from MATLAB. Since the MinGW support of MATLAB is somewhat limited, you will probably have to define the environment variable ``MW_MINGW64_LOC`` and assign the path of your MinGW installation. Hint to avoid misunderstandings: The last-mentioned directory contains folders named ``bin``, ``include``, and ``lib`` among others. Then, the following command should generate the required MEX-file::
+
+  make matlab CPATH=(PATH_OF_FFTW_H) LIBRARY_PATH=(PATH_OF_FFTW(F)_DLL) MFLAGS='-largeArrayDims -DR2008OO -L(PATH_OF_LGOMP_DLL) -lgomp.dll -L(PATH_OF_FFTW(F)_DLL)'
```

Comment:
Again, would be neater to add to make.inc:
MFLAGS += -L(PATH_OF_LGOMP_DLL) -lgomp.dll -L(PATH_OF_FFTW(F)_DLL)
Actually the FFTW DLL could be referred to from before.

Reply:
The new approach should take care of this issue